iOS/ViewController.mm

```diff
@@ -124,7 +128,11 @@ - (void)viewDidLoad {
     mainTitle.text = [NSString stringWithFormat:@"Superpowered Latency Test v%@", [[NSBundle mainBundle] objectForInfoDictionaryKey:@"CFBundleShortVersionString"]];
     if ([[AVAudioSession sharedInstance] respondsToSelector:@selector(recordPermission)] && [[AVAudioSession sharedInstance] respondsToSelector:@selector(requestRecordPermission:)] && ([[AVAudioSession sharedInstance] recordPermission] != AVAudioSessionRecordPermissionGranted)) {
         [[AVAudioSession sharedInstance] requestRecordPermission:^(BOOL granted) {
-            if (granted) [self finishLoading];
+            if (granted) {
+                dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
I'd prefer a `performSelectorOnMainThread` here. It's just a cosmetic change, but a single liner.